cli/src/test/resources/ide-projects/jmc/downloadMockLocation/org.openjdk.jmc-8.3.0-linux.gtk.x86_64/JDK Mission Control/jmc

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "Dummy jmc 8.3.0 on linux" > jmcTestRestult.txt
```

Comment:
I guess that when we just echo, we cannot easily get the result that was echoed in the test and assert it so most probably this is the easiest working solution.
However, please always also echo `$@` in such dummy commandlets so we can test the passed parameters (including parameters potentially added by the commandlet - see all the magic in eclipse that we can then test in such way).